db/migrate/20190313142540_add_shipping_category_to_shipping_methods_and_products.rb

```diff
@@ -0,0 +1,13 @@
+class AddShippingCategoryToShippingMethodsAndProducts < ActiveRecord::Migration
+  def up
+    # This is different from the equivalent Spree migration
+    #   Here we are creating the default shipping category even if there are already shipping categories
+    default_category = Spree::ShippingCategory.create!(name: "Default")
+
+    Spree::ShippingMethod.all.each do |method|
```

Comment:
I know this comes from Spree and changing it is out of scope. But just for similar cases in the future `Spree::ShippingMethod.find_each` would be better as it iterates in batches when there are lots of records. https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each

It only matters if there are more than 1000 records and we have ~500 in Australia. So it should be fine.

Reply:
nice, thanks for pointing that out!

Reply:
that applies to all updates we perform. I checked the one SetDefaultStockLocationOnShipments and I don't think it'll be a problem for the numbers we have so I prefer to lean towards code simplicity instead of performance. Agree on this?

Reply:
:+1: